src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -1,7 +1,43 @@
+require "y2storage"
 require "yast"
 require "cwm/dialog"
 require "ui/greasemonkey"
 
+Yast.import "Popup"
+
+module Y2Storage
+  # Monkey-patched Region to get a size
```

Comment:
do you discuss it with ancorgs? maybe it make sense to have it in yast2-storage itself. and do not consider refinements instead of monkey patching if it is not possible otherwise?